packages/graphql-client/src/graphql/login.mutation.js

```diff
@@ -5,14 +5,13 @@ export const createLoginMutation = userFieldsFragment => gql`
   mutation($user: String!, $password: String!) {
     loginWithPassword(user: $user, password: $password) {
       sessionId
+      ...LoginFields
       user {
-        id
```

Comment:
I think we should try to ask for `id` regardless of what user fragment we get later